ts/packages/library/lib/index.cjs

```diff
@@ -35704,8 +35704,169 @@ class CandidService {
     constructor() { }
 }
 
+const defaultChannelOptions = {
+    isPublic: true,
+    permissions: {
```

Comment:
The permissions field is optional so you could pass in None to get default. But I suppose you want to illustrate the structure to devs?

Reply:
I didn't know that None would give me the defaults. It certainly seems clearer to pass in the defaults that you want (to me). And also it makes it easier to _tweak_ the defaults if you have them available.